### PR TITLE
Revert to two-step process for deploying Tekton

### PR DESCRIPTION
**Pull request by blancadesal**

### CODE REVIEW

**Comment by david-caro** [comment]
I think you will need to remove the base-tekton directory from:
https://github.com/toolforge/buildservice/blob/main/deploy/toolsbeta/kustomization.yaml#L7

for both toolsbeta and tools

File: README.md [hunk 1]
```diff
@@ -54,6 +46,14 @@ You can install Harbor with the helper script:
 After that you can use docker-compose to run the whole harbor system:
 - `docker-compose -f .harbor/harbor/docker-compose.yml up -d`
 
+Once Harbor is up and running (see the section "Setup Harbor" below), you can
```
Comment (david-caro) [inline README.md:49]: This will need rewording now, not sure if this change was intentional (seems a bit unrelated).

Might be interesting to keep though a note before too, something like "we will be able to test if that choice works after harbor is up".

Comment (blancadesal) [reply README.md]: You are right that this does not really belong here, so I've dropped the last commit and will do a more comprehensive edit of the README separately.

**Comment by david-caro** [comment]
LGTM!
